Tests/LootLockerTests/PlayMode/LeaderboardTest.cs

```diff
@@ -0,0 +1,495 @@
+using LootLocker;
+using LootLocker.Requests;
+using LootLockerTestConfigurationUtils;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Random = UnityEngine.Random;
+
+namespace LootLockerTests.PlayMode
+{
+    public class LeaderboardTest
```

Comment:
I don't remember what the ticket said, but I'm missing tests around pagination and GetMemberRank etc.